pkg/consensus/consensus.go

```diff
@@ -0,0 +1,32 @@
+package consensus
+
+import "github.com/CityOfZion/neo-go/pkg/util"
+
+const cacheMaxCapacity = 100
+
+// Service represents consensus instance.
+type Service interface {
+	OnPayload(p *Payload)
+	GetPayload(h util.Uint256) *Payload
+}
+
+type service struct {
+	cache *relayCache
+}
+
+// NewService returns new consensus.Service instance.
+func NewService() Service {
+	return &service{
+		cache: newFIFOCache(cacheMaxCapacity),
```

Comment:
btw why we choose this caching mechanism? why not LFU?

Reply:
Payloads will generally be removed in FIFO order: we no longer need payloads from last epoch after a block has been accepted.